vm/devices/storage/disk_nvme/src/lib.rs

```diff
@@ -16,17 +16,19 @@ use nvme_spec::nvm;
 use nvme_spec::Status;
 use pal::unix::affinity::get_cpu_number;
 use std::io;
+use std::sync::Arc;
 
 #[derive(Debug, Inspect)]
 pub struct NvmeDisk {
+    /// NVMe namespace mapped to the disk representation.
     #[inspect(flatten)]
-    namespace: nvme_driver::Namespace,
+    namespace: Arc<nvme_driver::Namespace>,
     #[inspect(skip)]
     block_shift: u32,
 }
 
 impl NvmeDisk {
-    pub fn new(namespace: nvme_driver::Namespace) -> Self {
+    pub fn new(namespace: Arc<nvme_driver::Namespace>) -> Self {
```

Comment:
Do we still need this to be `Arc`?

Reply:
Removed Arc<>, restored the original code.